### DISCUSSION TITLE
Multi Timezone

### DISCUSSION THREAD

**Discussion by reach2rv** [serenity-is/Serenity#7079] [Help] [answered]
@VictorTomaili @volkanceylan 

How serenity handles multiple timzones for users. For ex. If one user is entering data in UTC +3 where as other user accessing same data in UTC + 5

**Comment by VictorTomaili** [comment, marked as answer]
Save date times on database with `DateTimeKind.UTC`
When a data is updated in UTC+5 timezone, this data should be available in UTC+5 time zone immediately. You cannot wait for UTC+3 time to match.
On script side, the dates should be automatically shown in correct date time because of the browser date time zone. It converts utc time to the local time zone. See this [format function](https://github.com/serenity-is/Serenity/blob/master/packages/base/src/formatting.ts#L644)
